docs/admin-manual/configuration-macros.rst

```diff
@@ -11404,3 +11404,20 @@ condor_annex Configuration File Macros
 
 See :doc:`/cloud-computing/annex-configuration` for :tool:`condor_annex`
 configuration file macros.
+
+``htcondor annex`` Configuration File Macros
+--------------------------------------------
+:index:`htcondor annex configuration variables<single: htcondor annex configuration variables; configuration>`
+
+:macro-def:`HPC_ANNEX_ENABLED`
+    If true, users will have access to the ``annex`` noun of the
+    :doc:`../man-pages/htcondor` command.
+
+    .. warning::
+
+        This does not configure the AP so that ``htcondor annex``
+        will *work*.  Configuring an AP for ``htcondor annex`` is
+        tricky, and we recommend that you add :macro:`use feature:HPC_ANNEX`
+        instead, which sets this macro.
+
+    Defaults to false.
```

Comment:
Can we add the following knobs to the big list of configuration macros on this page, as they are referenced elsewhere, but not macro-def'd here:

ANNEX_DEFAULT_AWS_REGION'
'ANNEX_DEFAULT_ODI_INSTANCE_PROFILE_ARN'
'ANNEX_DEFAULT_SFR_CONFIG_FILE'
'ANNEX_DEFAULT_CF_URL'

Reply:
Those aren't for this annex; do you want to mix them into this PR anyway?

Reply:
https://htcondor.readthedocs.io/en/latest/cloud-computing/annex-configuration.html#ANNEX_DEFAULT_AWS_REGION
https://htcondor.readthedocs.io/en/latest/cloud-computing/annex-configuration.html#ANNEX_DEFAULT_ODI_INSTANCE_PROFILE_ARN
https://htcondor.readthedocs.io/en/latest/cloud-computing/annex-configuration.html#ANNEX_DEFAULT_SFR_CONFIG_FILE
https://htcondor.readthedocs.io/en/latest/cloud-computing/annex-configuration.html#ANNEX_DEFAULT_CF_URL

Reply:
Tell Cole to fix his script. ;)

Reply:
(Cole will fix his script, probably fairly shortly, but that doesn't need to block this PR now that main's been merged back into it and the number of doc warnings is so reduced.)